channeldb/db_test.go

```diff
@@ -33,7 +33,7 @@ func TestOpenWithCreate(t *testing.T) {
 
 	// Next, open thereby creating channeldb for the first time.
 	dbPath := filepath.Join(tempDirName, "cdb")
-	backend, cleanup, err := kvdb.GetTestBackend(dbPath, "cdb")
+	backend, cleanup, err := kvdb.GetTestBackend(dbPath, dbName)
```

Comment:
Good catch, what about also reopening the sql database ? Adopting the `Open(dbPath, OptionDryRunMigration(true))` function for the sql backend.

Reply:
Not really, I think when trying to reopen the db we should not default on open with BBolt but also reopen the sqlite case ?

https://github.com/lightningnetwork/lnd/blob/master/channeldb/db.go#L342-L355

Reply:
Oh right, so `GetTestBackend` will open with the configured backend (either sqlite, bolt or postgres). Then `CreateWithBackend` will use that backend to create the chandb.